src/events/DistanceEvents.ts

```diff
@@ -1,10 +1,10 @@
-export interface IPosition {
+export interface IGeolocationPosition {
```

Comment:
Since this is referenced in different jsdoc comments, this interface should be documented.

+ Since you need it for your external code, documenting it will help us prevent breaking changes.